### DISCUSSION TITLE
CoPilot Will Not Sign In (VSCode)

### DISCUSSION THREAD

**Discussion by ed-p-may** [community/community#128294] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

CoPilot will not sign in, chat panel is not displayed, not working. 

Installs fine, status shows 'ready':

```
2024-06-12 17:04:56.637 [info] [fetcher] Using Helix fetcher.
2024-06-12 17:04:56.637 [info] [code-referencing] Public code references are enabled.
2024-06-12 17:04:56.649 [info] [auth] Sucessfully authenticated
```

but in CoPilot Chat, it only shows a blue 'Sign In' button. Button is non-response. Not error, no message, no anything. 
![Screenshot 2024-06-12 at 5 07 06 PM](https://github.com/community/community/assets/69652712/44eb1a01-7a99-4c0f-a0e7-8ee2690bf1b8)

Plugin un-installed, re-installed several times. System rebooted.

## Environment:
- Apple M1 Max
- MacOS Ventura 13.6.6
- VSCode 1.90.0
- GutHub CoPilot v1.201.0

@ed-p-may

**Comment by davevad93** [comment]
Hi @ed-p-may ,

If you haven't tried this yet, try logging out from Copilot in VS Code and then logging back in.

**Reply by davevad93** [reply]
The output shows `Sucessfully authenticated` so I guess you should be able to `sign out` by clicking on the account icon.

**Reply by ed-p-may** [reply]
Thank you @davevad93 , unfortunately I do not see any options for sign out
![Screenshot 2024-06-12 at 5 49 44 PM](https://github.com/community/community/assets/69652712/5dbdd8df-5e2c-4d8a-949c-014f25180ea0)

copilot itself seems to work (inline code completion) but just not chat sign-in?
![Screenshot 2024-06-12 at 5 51 33 PM](https://github.com/community/community/assets/69652712/54668f17-d805-4c79-bff8-cd20ff13fb49)

perhaps its just another GH issue. Actions were down for an hour yesterday, maybe its just more of that?

**Reply by davevad93** [reply, marked as answer]
You're welcome @ed-p-may , If you were using the chat normally up to now, It could be indeed a GitHub issue, however my Copilot chat in `VS Code` is working fine with `Windows`, no strange issues so far. About the sign out thing, an account icon like the following should appear on the bottom left corner:

![0](https://github.com/community/community/assets/94308234/b95a85de-7266-40a3-87dc-d92e9131df8e)

**Reply by ed-p-may** [reply]
ah! that was it. back to normal now. Thank you!
@ed-p-may

**Reply by davevad93** [reply]
You're welcome, I'm glad it's fixed!

**Reply by ArcTens** [reply]
same issue here, it was logged out of everything and had no clue why.  I had multiple github accts logged in which didnt cause an issue until the past 24 hours when I was trying to deploy and it was logged into both somehow and using the one not connected to co-pilot.   

IMO if logged into multiple accounts the github chat screen should request to select acct instead of having a dead login button that does nothing

**Reply by savar33** [reply]
I have the same issue, on MacOS.  
I use only one github account in the VSCode. 
Signing out and signing in back didn't fix the issue for me

**Reply by chrvsd** [reply]
@savar33 I still have not been able to get my VSCode Chat on MacOS personal M1 Pro working either using an individual plan. Nothing seemed to have worked for me either :(

**Reply by savar33** [reply]
@chrvsd right. BTW the same github account works in vscode in windows 11 for me.

**Reply by chrvsd** [reply]
@savar33 I updated VSCode to this version and it works finally! I did not have to do anything special, just relogin.

Version: 1.93.1
Commit: 38c31bc77e0dd6ae88a4e9cc93428cc27a56ba40

**Reply by savar33** [reply]
It worked for me as well. Thank you!
